components/SidePage/Suggestions/Suggestions.jsx

```diff
@@ -90,7 +88,7 @@ const GuideSuggestion = ({ guideSuggestion }) => {
     )
 }
 
-export const Suggestions = ({ guideSuggestions }) => {
+export const Suggestions = ({ guideSuggestions, currentQuery, isLoading }) => {
```

Comment:
suggestions for simplifying variable names:

- `guideSuggestions` -> `items` (we know that are suggestions)
- `currentQuery` -> `query` (simplify it a bit)